js/blocks/controls/post.js

```diff
@@ -1,15 +1,7 @@
-import ContentControl from './content';
+import { ContentControl } from '../components';
 
-const BlockLabPostControl = ( props, field, block ) => {
-	/**
-	 * Gets the post title from an API response.
-	 *
-	 * @param {Object} apiResponse The API response in which to look for the post title.
-	 * @return {String} The post title from the response, or the default.
-	 */
-	const getTitleFromAPI = apiResponse => ( apiResponse && apiResponse.title && apiResponse.title.rendered ) ? apiResponse.title.rendered : '';
-
-	return ContentControl( props, field, getTitleFromAPI );
+const BlockLabPostControl = ( props ) => {
```

Comment:
It looks like `getTitleFromAPI()` wasn't actually used in the `ContentControl`, or anywhere else. I'm not sure why I added it 😄